packetizer.go

```diff
@@ -39,10 +39,12 @@ func (p *packetHandler) getMessage(l int) (err error) {
 	// https://github.com/msgpack/msgpack/blob/master/spec.md#formats-array
 	// . Do this so we can decode directly into the expected
 	// fields without copying.
+	// We need this because the target type information isn't known until
```

Comment:
This comment sounds like a replacement for the previous sentence. Perhaps merge them to have "We need this ... first few fields, and we want to decode directly ... without copying."?